crypto/fipsmodule/rand/new_rand_test.cc

```diff
@@ -0,0 +1,43 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#include <gtest/gtest.h>
+
+#include <openssl/ctrdrbg.h>
+
+#include "new_rand_internal.h"
+
+
+// TODO
+// Remove when promoting to default
+#if !defined(BORINGSSL_PREFIX)
+
+#define COMPILATION_UNIT_NR_PREFIX
+#include "new_rand_prefix.h"
```

Comment:
Why not just have this file call the new random functions directly?

Reply:
See comment for `new_rand_prefix.h`.